src/src/targets/Eachine_TX526/Eachine_TX526.c

```diff
@@ -3,10 +3,8 @@
 #include "helpers.h"
 #include <Arduino.h>
 
-struct PowerMapping power_mapping[] = {
-  {0, 0}, {25, 14}, {200, 23},
-};
-uint8_t power_mapping_size = ARRAY_SIZE(power_mapping);
+
+uint8_t saPowerLevelsLut[SA_NUM_POWER_LEVELS] = {RACE_MODE, 1, 14, 23};
```

Comment:
"1" should be first in the array for Low power on Disarm.

Reply:
fixed.